src/chains/ethereum/block/package.json

```diff
@@ -56,13 +56,13 @@
     "typescript": "4.6.4"
   },
   "dependencies": {
-    "@ethereumjs/common": "2.6.4",
+    "@ethereumjs/common": "2.6.5",
     "@ganache/ethereum-address": "0.5.0",
     "@ganache/ethereum-transaction": "0.5.0",
     "@ganache/ethereum-utils": "0.5.0",
     "@ganache/rlp": "0.5.0",
     "@ganache/utils": "0.5.0",
     "emittery": "0.10.0",
-    "ethereumjs-util": "7.1.4"
+    "ethereumjs-util": "7.1.5"
```

Comment:
`@ethereumjs/*` packages have updated to this version of `ethereumjs-util`, so I matched our dependency with theirs so we don't have to install multiple.